flake.nix

```diff
@@ -6,7 +6,7 @@
     systems.url = "github:nix-systems/default";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/3929c54c3bb303c86a84628bcdff5c963fcbd037";
```

Comment:
`shared-kernel` should be updated without changing `flake.nix`. See https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs